packages/mold.rb

```diff
@@ -46,5 +46,10 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    # See https://github.com/rui314/mold/issues/697
+    return unless ARCH == 'x86_64'
```

Comment:
`if ARCH == 'x86_64'` would be clearer.

Reply:
This was a rubocop stylistic suggestion. I switched it from the `if` format.

Reply:
I guess it doesn't matter.  Just a bit more confusing.